packages/connect/src/protocol-connect/handler-factory.ts

```diff
@@ -327,7 +328,7 @@ function createStreamHandler<I extends Message<I>, O extends Message<O>>(
     );
     return {
       ...uResponseOk,
-      body: outputIt,
+      body: await untilFirst(outputIt),
```

Comment:
Not pretty, but I'm not sure how we could improve this.

Reply:
suggestion: I'm not sure if this suggestion would be strange for this codebase but it'd be nice to leave a comment here (or somewhere? maybe the untilFirst JSDoc itself?) explaining why this is necessary and maybe some detail about how node's behavior differs from something like cloudflare workers.

Reply:
Added explanatory comments in https://github.com/bufbuild/connect-es/pull/588/commits/6c6dce8fd0dea0acdc633175900224bbea49c711.